server/tests-py/test_scheduled_triggers.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python3
+
+import pytest
+from datetime import datetime
+from datetime import timedelta
+from croniter import croniter
+from validate import validate_event_webhook,validate_event_headers
+from _queue import Empty
+import time
+
+def stringify_datetime(dt):
+    return dt.strftime("%Y-%m-%dT%H:%M:%S.%fZ")
+
+def get_events_of_scheduled_trigger(hge_ctx,trigger_name):
+    events_count_sql = '''
+    select count(*) from hdb_catalog.hdb_scheduled_events where name = '{}'
+    '''.format(trigger_name)
+    q = {
+        "type":"run_sql",
+        "args":{
+            "sql":events_count_sql
+        }
+    }
+    return hge_ctx.v1q(q)
+
+@pytest.mark.usefixtures("evts_webhook")
+class TestScheduledTrigger(object):
+
+    cron_trigger_name = "a_scheduled_trigger"
+    adhoc_trigger_name = "adhoc_trigger"
+    cron_schedule = None
+    init_time = None
+    webhook_payload = {"foo":"baz"}
+    webhook_path = "/hello"
+    retries = 12
+    interval_in_secs = 5.0
+
+    @classmethod
+    def dir(cls):
+        return 'queries/scheduled_triggers'
+
+    def test_create_schedule_triggers(self,hge_ctx,evts_webhook):
+        q = {
+            "type":"run_sql",
+            "args":{
+                "sql":"set time zone 'UTC'"
+            }
+        }
+        st,resp = hge_ctx.v1q(q)
+        assert st == 200,resp
+        next_min = (datetime.utcnow() + timedelta(minutes=1)).minute
```

Comment:
can you get `datetime.utcnow()` out into a variable so that the value doesn't change between line 51 and 52.

Reply:
Yes, I'll make that change